leveldb_vendor/CHANGELOG.rst

```diff
@@ -0,0 +1,5 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package leveldb_vendor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+0.0.1 (2020-10-30)
+------------------
```

Comment:
please refrain from adding changelog files by hand. The release tool will take care of it.

Reply:
> please refrain from adding changelog files by hand. The release tool will take care of it.  

Okay.  
I will remove this file.